next/pages/search.tsx

```diff
@@ -0,0 +1,148 @@
+import "@elastic/react-search-ui-views/lib/styles/styles.css";
+
+import { GetStaticPropsResult } from "next";
+import Head from "next/head";
+import { useRouter } from "next/router";
+import { useTranslation } from "next-i18next";
+import { serverSideTranslations } from "next-i18next/serverSideTranslations";
+import React from "react";
+import {
+  ErrorBoundary,
+  Facet,
+  Paging,
+  PagingInfo,
+  Results,
+  SearchBox,
+  SearchProvider,
+  WithSearch,
+} from "@elastic/react-search-ui";
+import { Layout, LayoutProps } from "components/layout";
+import { SearchBoxInput } from "components/search/search-box-input";
+import MultiCheckboxFacet from "components/search/search-multicheckbox-facet";
+import PagingInfoView from "components/search/search-paging-info";
+import { SearchResult } from "components/search/search-result";
+import { getMenus } from "lib/get-menus";
+import buildRequest from "lib/search-ui-helpers/buildRequest";
+import buildState from "lib/search-ui-helpers/buildState";
+import runRequest from "lib/search-ui-helpers/runRequest";
+import { setLanguageLinks } from "lib/utils";
+
+import { LangContext } from "./_app";
+
+/**
+ * Contains the search provider component.
+ */
+export default function SearchPage({ menus }: LayoutProps) {
+  const { t } = useTranslation("common");
+  const router = useRouter();
+  const config = {
+    debug: false,
+    hasA11yNotifications: true,
+    apiConnector: null,
+    onSearch: async (state) => {
+      const { resultsPerPage } = state;
+      const requestBody = buildRequest(state);
+      const responseJson = await runRequest(requestBody, router.locale);
+      return buildState(responseJson, resultsPerPage, state);
+    },
+  };
+  return (
+    <LangContext.Provider
+      value={{
+        languageLinks: setLanguageLinks([]),
+      }}
+    >
+      <Layout menus={menus}>
+        <Head>
+          <title>Next.js for Drupal</title>
+          <meta
+            name="description"
+            content="A Next.js site powered by a Drupal backend."
+          />
+        </Head>
+
+        <h1 className="mb-10 text-6xl font-black">{t("search")}</h1>
+
+        <main>
+          <SearchProvider config={config}>
+            <WithSearch
+              mapContextToProps={({ wasSearched, results }) => ({
+                wasSearched,
+                results,
+              })}
+            >
+              {({ wasSearched, results }) => (
+                <ErrorBoundary>
+                  <div className="search-ui">
+                    <SearchBox
+                      searchAsYouType={false}
+                      shouldClearFilters={false}
+                      // Here we specify out own custom
+                      // component to render the search bar:
+                      inputView={SearchBoxInput}
```

Comment:
We are providing our custom component to render the view of the search bar. We have repeated this for all visible components